plugins/woocommerce/includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -149,7 +149,7 @@ protected function get_settings_for_default_section() {
 				'title'    => __( 'Logo width (px)', 'woocommerce' ),
 				'id'       => 'woocommerce_email_header_image_width',
 				'desc_tip' => '',
-				'default'  => 120,
+				'default'  => '120',
 				'type'     => 'number',
```

Comment:
shouldnt that be string then?

Reply:
This is used for `<input type="HERE">` - I used number so there is HTML validation for this being a number and also it adds the arrows to the input and allows changing the value with arrows. 
<img width="1051" alt="Screenshot 2025-04-04 at 09 59 06" src="https://github.com/user-attachments/assets/71d360e0-af2f-48a1-af35-559229fa081a" />

Reply:
So that type field is used to determine the input type?

Reply:
Yes. So you can also use anything there, e.g. 'my_custom_component` and render whatever there. Or you can use any of the predefined types, which will render the setting row for you https://github.com/woocommerce/woocommerce/blob/6376d46d9c2aca9d097f4a37669e032424094818/plugins/woocommerce/includes/admin/class-wc-admin-settings.php#L333-L367